### ISSUE TITLE
put-916805303: permission denied

### ISSUE THREAD

**Issue by jbenet** [ipfs/go-ipfs#1877]
```
ERRO[06:55:17:000] open /home/jbenet/.ipfs/blocks/1220e011/put-916805303: permission denied  module=commands/http
```
from a long add. halts

**Comment by whyrusleeping** [comment]
could I  get the output of `ipfs diag sys`?

**Comment by jbenet** [comment]
```json
ipfs diag sys
{
  "diskinfo": {
    "free_space": 3.259375616e+09,
    "fstype": "ext2/ext3",
    "total_space": 2.1003628544e+10,
    "used_space": 1.7744252928e+10
  },
  "environment": {
    "GOPATH": "",
    "IPFS_PATH": ""
  },
  "ipfs_git_sha": "",
  "ipfs_version": "0.3.8",
  "memory": {
    "swap": {
      "free": 0,
      "sin": 0,
      "sout": 0,
      "total": 0,
      "used": 0,
      "used_percent": 0
    },
    "virt": {
      "active": 3.702784e+08,
      "available": 1.40734464e+08,
      "buffers": 1.7698816e+07,
      "cached": 7.6611584e+07,
      "free": 4.6424064e+07,
      "inactive": 4.6292992e+07,
      "shared": 0,
      "total": 5.1386368e+08,
      "used": 4.67439616e+08,
      "used_percent": 72.6124905344546,
      "wired": 0
    }
  },
  "net": {
    "interface_addresses": [
      "/ip4/127.0.0.1",
      "/ip4/104.236.32.22",
      "/ip4/172.17.42.1",
      "/ip6/::1",
      "/ip6/fe80::601:2fff:fe27:2801",
      "/ip6/fe80::42:66ff:fe1c:571a"
    ]
  },
  "runtime": {
    "arch": "amd64",
    "compiler": "gc",
    "gomaxprocs": 3,
    "numcpu": 1,
    "numgoroutines": 1439,
    "os": "linux",
    "version": "go1.5.1"
  }
}
```

**Comment by whyrusleeping** [comment]
havent seen this issue since, going to assume this was an OSX filesystem fluke. please reopen if it happens again.
